### DISCUSSION TITLE
Solar load using radiation model

### DISCUSSION THREAD

**Discussion by ananddoss2004** [blueCFD/Core#165] [Q&A] [answered]
Does blueCFD has solar load radiation model capabilities ? Also solar tracker algorithm ?

**Comment by wyldckat** [comment, marked as answer]
blueCFD-Core 2017 provides OpenFOAM 5.x: http://bluecfd.github.io/Core/ReleaseNotes/bluecfd-core-2017-2/ - where said 5.x version is developed/provided by the OpenFOAM Foundation.

If you use Google to search for:

```
site:openfoam.org solar
```

there will be no occurrences.

On the other hand, if you search for:

```
site:openfoam.com solar
```

there will be several occurrences. OpenFOAM.com is OpenCFD-ESI, which currently is not able to provide features to OpenFOAM.org.

**Reply by wyldckat** [reply]
I've converted this issue into a Q&A topic at our new Discussions section here at the blueCFD-Core project.
